### DISCUSSION TITLE
Start modded game from a shortcut

### DISCUSSION THREAD

**Discussion by onoderis** [ebkr/r2modmanPlus#693] [Q&A] [answered]
Is is possible to start modded game from a shortcut?
Right now I have to launch r2modman, select profile and click "Start modded". It's a little bit annoying to repeat those actions every single time I run the game.

**Comment by ebkr** [comment, marked as answer]
The reason you have you start the manager is so that you can see if any mods have updates.

You can make your own shortcut to directly start the game with mods by looking at the launch parameters in the help section of the manager.

**Reply by specreck** [reply]
what in the fuck did you say
